packages/main/src/plugin/container-registry.ts

```diff
@@ -1878,11 +1878,9 @@ export class ContainerProviderRegistry {
       }
 
       const engine = this.internalProviders.get(engineId);
-      if (engine) {
+      if (engine && (options.start === true || options.start === undefined)) {
+        await container.start();
```

Comment:
AFAIK we need to attach before starting a container

Reply:
else you may not receive the output

Reply:
The reason for the switch is the following:

- [Docker documentation](https://docs.docker.com/reference/cli/docker/container/attach/) mentions attach is for running containers
- podman attach (CLI) rejects if container is not running
- current behavoir in Podman Desktop is attach API call never returns